cmake/mcuboot.cmake

```diff
@@ -136,6 +171,10 @@ function(zephyr_mcuboot_tasks)
     set_property(GLOBAL APPEND PROPERTY extra_post_build_commands COMMAND
       ${west_sign} ${confirmed_args} ${imgtool_args} --pad --confirm)
   endif()
+  if(encrypted_args)
+    set_property(GLOBAL APPEND PROPERTY extra_post_build_commands COMMAND
+      ${west_sign} ${encrypted_args} ${imgtool_args} --encrypt "${keyfile_enc}")
```

Comment:
Do we really need to call `west sign` twice? Or can we build a single `west sign` command that generates padded, confirmed, and encrypted images as needed? If a single command is possible, please do it that way to avoid an extra subprocess for every build.

Reply:
I didn't find a way to do that with a single command, given that I want both unencrypted and encrypted output. If anyone has a way of doing that, please let me know and I will try to use it.